apps/mobile/src/screens/quiz-entry/quiz-entry.tsx

```diff
@@ -35,12 +49,13 @@ const QuizEntry: React.FC = () => {
 					<Text preset="heading" style={styles.text} weight="extraBold">
 						Craft your personal{"\n"}Life Balance Wheel!
 					</Text>
-					<Image
-						source={
-							require("~/assets/images/balance-wheel.png") as ImageSourcePropType
-						}
-						style={styles.image}
-					/>
+					<View style={globalStyles.alignItemsCenter}>
```

Comment:
Just a nitpick, but let's leave here the image and use the actual wheel on the Wheel tab of bottom slider